uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-calendar-month.component.js

```diff
@@ -542,6 +577,13 @@ define([
       }
     }
 
+    function sortLeaveRequests (leaveRequests) {
```

Comment:
`sortLeaveRequestsByDate`

Reply:
Not really, not only date/time matter, but also if it is AM/PM.